### ISSUE TITLE
when i export my flowchart from the free open sourced draw.io to a visio file and upload it onto visio web, the arrows are not glued to the boxes and every amendment i make results in the rest of the arrows shifting to other parts of the page. how can i go about fixing this issue?

### ISSUE THREAD

**Issue by lenice-fs** [jgraph/drawio#2646]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version [e.g. 12.6.7]

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by mararad** [comment]
Which OS, browser version, Visio and diagrams.net version are you using? 
Is it possible to attach diagram file and print screen of file opened in Visio?
Was diagram created in diagrams.net initially?

P.Ss Export as VSDX is unfotunately still beta so not sure if we'll be able to help.

**Comment by lenice-fs** [comment]
Hello!

Here are the details you asked for:

OS: macOS Catalina 10.15.7
Browser version: Safari 14.1.2 (but my colleague tried on another browser
on her windows and we ran into the same problem)
Visio version: Visio web
Diagrams.net version: Not really sure so i attached a test file

Upon testing my small test file I realised that the shifting of the arrows
with each amendment doesn't actually happen. However the flowchart shapes
are still not recognised by Visio, I understand that it is a beta version
but is there any way I can fix it, would paying for the paid versions of
draw.io make the exporting to Visio smoother? Hope to hear from you soon!

Thanks,
Lenice Lim
